pygfx/renderers/wgpu/_mipmapsutil.py

```diff
@@ -54,155 +57,144 @@
 """
 
 
-class MipmapsUtil:
-    def __init__(self, device) -> None:
-        self.device = device
-
-        # Cache pipelines for every texture format used.
-        self.pipelines = {}
+def get_mip_level_count(texture):
+    """Get the number of mipmap levels from the texture size."""
+    assert isinstance(texture, Texture)
```

Comment:
I think you can guess what I have to say about `assert` :)

Reply:
Sorry 😄